third-party/crowdsec/src/main/java/org/apache/james/model/CrowdsecClientConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.model;
+
+import java.net.URL;
+import java.util.Optional;
+
+import org.apache.commons.configuration2.Configuration;
+
+import com.github.fge.lambdas.Throwing;
+
+public class CrowdsecClientConfiguration {
+    public static final Integer DEFAULT_TIMEOUT_IN_SECONDS = 15;
```

Comment:
IMO let's reduce the timeout for CrowdSec: if fail, fail fast.

The reason is CrowdSec seems to be placed on the surface near users (SMTP EHLO, IMAP connection...). CrowdSec slowness or failure should not affect much on users' experience.

While Rspamd is more on background work which could rely on longer timeout.

Reply:
Okay, I got it

Reply:
Strong typing: use `Duration` as a type